stable/mlrun/templates/api-role.yaml

```diff
@@ -1,18 +0,0 @@
-{{- if .Values.rbac.create -}}
```

Comment:
By doing that you're forcing anyone using this chart to use (which means they need permission to create) ClusterRole
I suggest leaving the option to work using the "old" way - with simple `Role`:
* Create Role when
`namespacedResources.create` is True and no additional namespaces
* Create ClusterRole when  `clusterResources.create` is True